binaries/scheduler/config/local.memory

```diff
@@ -5,7 +5,9 @@
   },
   "SchedulerConfig": {
     "Type": "stateful",
-    "MaxRetriesPerTask" : 2
+    "MaxRetriesPerTask" : 2,
+    "DebugMode" : false,
+    "RecoverJobsOnStartup" : false
```

Comment:
The JSON parser actually throws an error if there is a comma on the last element

"Error Parsing Config File local.local, with Error Couldn't parse top-level config: invalid character '}' looking for beginning of object key string"